### ISSUE TITLE
Simple error

### ISSUE THREAD

**Issue by FPtje** [Nayruden/Ulysses#483]
Started a listen server with ulx cleanly installed. MOTD was empty and this showed up:

```

[ERROR] addons/ulx/lua/ulx/modules/cl/motdmenu.lua:29: bad argument #1 to 'SetHTML' (string expected, got nil)
  1. SetHTML - [C]:-1
   2. fn - addons/ulx/lua/ulx/modules/cl/motdmenu.lua:29
    3. func - addons/ulib/lua/ulib/client/cl_util.lua:22
     4. unknown - lua/includes/extensions/net.lua:32

```

Solution: Change line 29 of that file to
```lua
html:SetHTML( ULib.fileRead( "data/ulx_motd.txt" ) or "" )
```

**Comment by 00875** [comment]
https://github.com/Nayruden/Ulysses/pull/471

**Comment by Nayruden** [comment]
Fixed via d2de326ddf897c3d5ec0e78956ec74c4acd1887e
